core/sodasql/cli/cli.py

```diff
@@ -62,6 +66,23 @@ def create(warehouse_type: str,
 
     WAREHOUSE_TYPE is one of {postgres, snowflake, redshift, bigquery, athena}
     """
+
+    soda_telemetry.set_attribute('cli_command_name', 'create')
+    soda_telemetry.set_attribute('datasource_type', warehouse_type)
+
+    span_setup_function_args(
+        {
+            'command_argument': {'datasource_type': warehouse_type},
+            'command_option':
+                {
+                    'file': file,
+                    'warehouse': warehouse,
+                    'database': database,
+                    'username': username
```

Comment:
Is this the username or whether they pass a username?

Reply:
it is the username you can pass that will be added to the warehouse.yml

Reply:
yup it was that, I changed warehouse, database and username to be just booleans whether the param has been passed and renamed it to `using_X`